### ISSUE TITLE
Example code is making query to database

### ISSUE THREAD

**Issue by fccoelho** [thegraphnetwork/epigraphhub_py#166]
## 🐛 Bug Report

In the example code related to the Swiss covid forecast, which will be referenced by our soon-to-be published paper, there is acall that is trying to access the Epigraphhub database. This code should rely exclusively on the CSVs provided in the examples folder, so that anyone without access to the database can run the code.

## 🔬 How To Reproduce

The error can be reproduced by running the example on the `result_article_forecast` notebook.

here are the relevant pieces of the traceback:

```python
File ~/Documentos/Software_projects/epigraphhub_py/epigraphhub/models/forecast_swiss.py:327, in train_single_canton(target_curve_name, canton, predictors, path, vaccine, smooth, ini_date, end_date, parameters_model, predict_n, look_back)
     target_name = f"{target_curve_name}_{canton}"
     # getting the data
--> df = get_cluster_data(
           "switzerland", predictors, list(cluster_canton), vaccine=vaccine, smooth=smooth
     )
     # filling the nan values with 0
    df = df.fillna(0)
```
if the access to the database is not active it then fails.

**Comment by eduardocorrearaujo** [comment]
It was solved by the config done by Ivan. (PR #169)

**Comment by eduardocorrearaujo** [comment]
It was solved in the PR #167.
